arches/app/views/api/__init__.py

```diff
@@ -2383,3 +2383,128 @@ def delete(self, request, identifier=None):
                 status=400,
             )
         return JSONResponse(status=204)
+
+
+class UserPreference(APIBase):
+    def get(self, request, identifier=None):
+        """
+        Returns specific user preference when given uuid
+        If no id specified, and depending on user permissions, returns a list of all user preferences
+        """
+
+        # check user permission
+        overall_access = False
+        if request.user.groups.filter(name="Application Administrator").exists():
```

Comment:
I think we have to start using the permissions framework for this, unless/until archesproject/arches#11739

Reply:
(never mind about #11739, that wouldn't help because we're not using has_perm() here, but my understanding is we need to find the right util from arches.app.utils.permission_backend to call here)

Reply:
Checked with @aarongundel and I was mistaken -- it's fine to check Django groups here. The permissions framework doesn't replace / stand in for django user groups.